middle_end/semantics_of_primitives.ml

```diff
@@ -142,6 +142,10 @@ let for_primitive (prim : Lambda.primitive) =
     Misc.fatal_errorf "The primitive %a should have been eliminated by the \
         [Closure_conversion] pass."
       Printlambda.primitive prim
+  | Pload8
+  | Pload16 _
+  | Pload _ ->
+    No_effects, Has_coeffects
```

Comment:
@alainfrisch It is not used that way yet, but effectively this is something to consider. Currently it is only used to allow to remove the expression.
As this problem would matter for other unsafe primitives too, this is something that will change on all of them at the same time.